samples/BoltOn.Samples.WebApi/Startup.cs

```diff
@@ -22,23 +23,27 @@ public Startup(IConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
-			services.BoltOn(options =>
-			{
-				options.BoltOnEFModule();
-				options.BoltOnAssemblies(typeof(PingHandler).Assembly, typeof(StudentRepository).Assembly);
-			});
-
-			services.AddDbContext<SchoolDbContext>(options =>
-			 {
-				 options.UseSqlServer("Data Source=127.0.0.1;initial catalog=Testing;persist security info=True;User ID=sa;Password=$Password1;");
-			 });
-		}
+            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_2);
+            services.BoltOn(options =>
+            {
+                options.BoltOnEFModule();
+                options.BoltOnCosmosDbModule();
+                options.BoltOnAssemblies(typeof(PingHandler).Assembly, typeof(StudentRepository).Assembly);
+            });
+
+            services.AddDbContext<SchoolDbContext>(options =>
+             {
+                 options.UseSqlServer("Data Source=127.0.0.1;initial catalog=Testing;persist security info=True;User ID=sa;Password=$Password1;");
+             });
+
+            services.Configure<CosmosDbSettings>(options => Configuration.GetSection("CosmosSettings").Bind(options));
```

Comment:
what if we retrieve the CosmosSettings from appSettings and set it in BoltOnCosmosDbModule()? will that work?

Reply:
@gokulm : even the EF context we are registering the same way. I guess this change would depend on things like whether we give the consumer of package the control of appsettings structure or we define the structure and also on which pattern we use for reading settings which we haven't decided yet i guess!